wireapp/kalium
# fix: New calls mute the ongoing call(AR-2025)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Incoming calls affect current call state, if you mute incoming call the current call will also be muted

### Causes (Optional)

we were calling muteCall method from AVS in incoming call screen

### Solutions

muteCall method from AVS should be called only for established calls
 
### Dependencies (Optional)

There is a fix also on Reloaded so this PR should be merged with
[fix: New calls mute the ongoing call(AR-2025)
](https://github.com/wireapp/wire-android-reloaded/pull/833)

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

- First you should be on an ongoing call and turn the microphone off
- Receive another call
- Un-mute the incoming call
- You should notice that the active call is un-muted as well

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
